test/suite/test_tiered03.py

```diff
@@ -41,41 +43,62 @@ class test_tiered03(wttest.WiredTigerTestCase):
     # sharing would probably need to be reworked.
     uri = 'file:test_tiered03'
 
+    storage_sources = [
+        ('local', dict(ss_name = 'local_store',
+            auth_token = get_auth_token('local_store'),
+            bucket = get_bucket1_name('local_store'),
+            bucket_region = get_bucket1_region('local_store'),
+            bucket_prefix = "pfx_")),
+        # WT-8896 - S3 extension doesnt gets stuck at initializing if more than one simultaneous
```

Comment:
I think this should be `FIXME-WT-8896` (there might even be a script in `dist` that looks for `FIXME` on a ticket). Also the sentence does not quite make sense. I suggest rewording the beginning of the sentence maybe like:
"The S3 extension gets stuck during initialization if more..."

Reply:
FYI, the script is [s_mentions](https://github.com/wiredtiger/wiredtiger/blob/develop/dist/s_mentions).